doc/release-notes/release-notes-5.3.1-rc1.md

```diff
@@ -0,0 +1,99 @@
+Notable changes
+===============
+
+Fixed
+-----
+
+This release fixes an error "Assertion `uResultHeight == rewindHeight` failed" (#5958)
+that could sometimes happen when restarting a node.
+
+Memory Usage Improvement
+------------------------
+
+The memory usage of zcashd has been reduced by not keeping Equihash solutions for all
+block headers in memory.
```

Comment:
We should add the `asOfHeight` RPC changes when we do the changelog for the v5.3.1 release.